clients/drcachesim/tracer/raw2trace.h

```diff
@@ -876,6 +882,13 @@ template <typename T> class trace_converter_t {
             DR_CHECK(reinterpret_cast<trace_entry_t *>(buf) == buf_base,
                      "We shouldn't have buffered anything before calling "
                      "append_bb_entries");
+            if (saw_filter_endpoint && true) {
+                int file_type = impl()->get_file_type(tls);
+                file_type &= ~(OFFLINE_FILE_TYPE_FILTERED | OFFLINE_FILE_TYPE_IFILTERED |
+                               OFFLINE_FILE_TYPE_DFILTERED);
+                file_type |= OFFLINE_FILE_TYPE_DEFAULT;
+                impl()->set_file_type(tls, (offline_file_type_t)file_type);
```

Comment:
Does this also change it in the post-processed trace? Or is it just for internal logic? Comment either way.

Reply:
Added comment.